src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6832,6 +6832,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureDictionaryExpressions" xml:space="preserve">
     <value>dictionary expressions</value>
   </data>
+  <data name="IDS_FeatureCollectionExpressionArguments" xml:space="preserve">
+    <value>collection expression construction arguments</value>
```

Comment:
>collection expression construction arguments [](http://example.com/codeflow?start=11&length=44)

"collection expression arguments"

Reply:
nit: Not sure if this is a separate feature than dictionary expressions.
We'll use a single test plan and branch. I would suggest merging speclets at some point (this way we have a single place to point customers to)

Reply:
perhaps.  but it's not really a dict-expr feature.  This applies for being able to pass collection-args to other CE types.

We could possibly merge this into `IDS_KeyValuePairElementsOrDictionaryTargetsOrWithElements`... but it seems easier to just have simpler IDSs with simple explanations what they refer to :)